app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
```

Comment:
- [ ] Kindly add show action in PostsController and It can be empty for this milestone.